src/components/PupilViews/PupilLessonOverview/PupilLessonOverview.view.tsx

```diff
@@ -48,6 +49,10 @@ export const PupilViewsLessonOverview = ({
     isLessonComplete,
   } = useLessonEngineContext();
   const subjectIconName: `subject-${string}` = `subject-${subjectSlug}`;
+  const [isMounted, setIsMounted] = useState(false);
+  useEffect(() => {
+    setIsMounted(true);
+  }, []);
```

Comment:
![rain-screaming](https://github.com/oaknational/Oak-Web-Application/assets/122096/89a643a1-2306-4ded-bef5-71b41c7f427f)

Reply:
hahahahahaha ! Finally I got you !